dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -527,37 +527,88 @@ private void handleEntry2tsv(List<String> args) throws ApiException, IOException
 
     /**
      * this function will check if the content of the file is CWL or not
+     * it will get the content of the file and try to find/match the required fields
```

Comment:
Ok, a good start. 
Maybe a bit more elaboration. For example, why does this return a Boolean rather than a boolean? (Is a null return possible and does it have any meaning? Put this under `@return`

In what conditions will this throw an exception or exit and in what cases does it return true/false. 

Thanks!